keepers/validator-keeper/src/lib.rs

```diff
@@ -195,13 +202,34 @@ pub async fn emit_validator_history_metrics(
         }
     }
 
-    let get_vote_accounts_count = get_vote_accounts_with_retry(client, MIN_VOTE_EPOCHS, None)
-        .await?
-        .len();
+
+    let get_vote_accounts = get_vote_accounts_with_retry(client, MIN_VOTE_EPOCHS, None)
+        .await?;
+
+    let get_vote_accounts_count = get_vote_accounts.len() as i64;
+
+    let vote_program_id = get_vote_program_id();
+    let alive_history_vote_account_count = get_multiple_accounts_batched(&all_history_vote_accounts, client)
+        .await
+        .expect("Cannot fetch all accounts")
+        .iter()
+        .filter_map(|account| account.as_ref()) // Filter out None and unwrap the Some values.
+        .filter(|account| vote_program_id == account.owner) // Filter accounts whose owner matches the vote_program_id.
```

Comment:
should be able to combine both filters into one

Reply:
Done